EnglishTranslation-master/ETC.tsv

```diff
@@ -21025,26 +21025,26 @@ ETC_20160310_021024	Blue Coral
 ETC_20160310_021025	Coral Wagon
 ETC_20160310_021026	Red Coral
 ETC_20160310_021027	Fallen Carnivore
-ETC_20160310_021028	Fedimian Warehouse Keeper
+ETC_20160310_021028	Fedimian Storage Keeper
 ETC_20160310_021029	Lump of Vines
 ETC_20160310_021030	Crumbling Vines
 ETC_20160310_021031	Elgos Monastery Vines
-ETC_20160310_021032	Convey Tales of Valor
-ETC_20160310_021033	Convey Local Information
+ETC_20160310_021032	Share tales of battle
+ETC_20160310_021033	Share stories of adventure
 ETC_20160310_021034	Jiangshi
 ETC_20160310_021035	Cheers
-ETC_20160310_021036	Discouragement
+ETC_20160310_021036	Discourage
```

Comment:
i saw this in-game and changed it to "Lament" on my end, i feel like it works better (and it fits)